docs/_includes/ds6/tooltip-custom.html

```diff
@@ -0,0 +1,157 @@
+<p>There are 3 types of tooltips available: traditional tooltips, infotips, and tourtips</p>
+<p>Tooltips pointer can be positioned using any one of the following elements based on the type (tooltip/infotip/tourtip): 
+    <span class="highlight">tooltip__arrow-top-left</span>, 
```

Comment:
Should we just make this a `<ul>` with `<li>`'s?

Reply:
With the spans for highlighting still, of course. Just giving a bulleted list might help our developers.